streams/de.m3u

```diff
@@ -376,8 +378,6 @@ https://sdn-global-live-streaming-packager-cache.3qsdn.com/26658/26658_264_live.
 https://h056.video-stream-hosting.de/easycast7-live/_definst_/mp4:livestreamhd20/playlist.m3u8?ref=
 #EXTINF:-1 tvg-id="SenderNeuJerusalem.de",Sender Neu Jerusalem (576p)
 https://bild-und-ton.stream/snj-de/snj-de.smil/.m3u8
-#EXTINF:-1 tvg-id="ServusTVDeutschland.de",ServusTV Deutschland (720p) [Geo-blocked]
-https://stv-live.akamaized.net/hls/live/2031012/lingeoSTVDEwebPri/master_3392.m3u8
```

Comment:
https://dms.redbull.tv/v5/destination/stv/stv-linear/personal_computer/http/at/de_AT/playlist.m3u8

https://stvlvstrm.redbullmediahouse.com/hls/live/2031011/lingeoSTVATwebPri/master.m3u8

Reply:
Found out afterwards but forgot to remove comment.